src/components/settingsModal.html

```diff
@@ -50,6 +50,7 @@ <h5 class="modal-title">Settings</h5>
                                 <tr data-bind="template: { name: 'BooleanSettingTemplate', data: Settings.getSetting('confirmBeformeMulchingAllPlots')}"></tr>
                                 <tr data-bind="template: { name: 'BooleanSettingTemplate', data: Settings.getSetting('showGymGoAnimation')}"></tr>
                                 <tr data-bind="template: { name: 'BooleanSettingTemplate', data: Settings.getSetting('disableRightClickMenu')}"></tr>
+                                <tr data-bind="template: { name: 'BooleanSettingTemplate', data: Settings.getSetting('showPokerusMapStatus')}"></tr>
```

Comment:
Can we move this setting to under the th? It's a part of the mapcolors

Reply:
Removed boolean setting